database/clicker.go

```diff
@@ -1,5 +1,114 @@
 package database
 
-func WriteDB() {
+import (
+	"bypass/getter"
+	"context"
+	"crypto/tls"
+	"crypto/x509"
+	"net"
+	"os"
+	"path"
+	"time"
 
+	"github.com/ClickHouse/clickhouse-go/v2"
+	"github.com/ClickHouse/clickhouse-go/v2/lib/driver"
+	"github.com/pkg/errors"
+	"gopkg.in/yaml.v3"
+)
+
+type Conf struct {
+	Database string `yaml:"database"`
+	Username string `yaml:"username"`
+	Password string `yaml:"password"`
+	Host     string `yaml:"host"`
+	Port     string `yaml:"port"`
+}
+
+type Row struct {
+	CreatedAt      time.Time `ch:"created_at"`
+	ProductID      string    `ch:"product_id"`
+	ID             int64     `ch:"id"`
+	SearchPhraseID int32     `ch:"search_phrase_id"`
+	SearchPosition int32     `ch:"search_position"`
+}
+
+func GetConfig() (*Conf, error) {
+	config := NewConf()
+	file, err := os.ReadFile("./files/conf.yaml")
+	if err != nil {
+		return config, errors.Wrap(err, ".yaml file not found")
+	}
+	if err = yaml.Unmarshal(file, &config); err != nil {
+		return config, errors.Wrap(err, ".yaml can't unmarshal")
+	}
+	return config, nil
+}
+
+func NewConf() *Conf {
+	return &Conf{
+		Database: "",
+		Username: "",
+		Password: "",
+		Port:     "",
+		Host:     "",
+	}
+}
+
+func WriteDB(writer <-chan getter.DataKeyword, errorCH chan error) {
+	for data := range writer {
+
+		conn, err := connect()
+		if err != nil {
+			errorCH <- errors.Wrap(err, "Can't connection")
+		}
+		ctx := context.Background()
+		batch, err := conn.PrepareBatch(ctx, "INSERT INTO default.product_search_position")
+		if err != nil {
+			errorCH <- errors.Wrap(err, "DB error")
```

Comment:
```suggestion
			errorCH <- errors.Wrap(err, "preparing batch")
```

при ошибке, писать ошибка не нужно) ату читать не очень понятно/красиво